monai/apps/pathology/transforms/post/array.py

```diff
@@ -691,4 +694,11 @@ def __call__(self, type_pred: NdarrayOrTensor, instance_pred: NdarrayOrTensor) -
                 inst_info_dict[inst_id]["type"] = inst_type  # type: ignore
                 inst_info_dict[inst_id]["type_probability"] = type_prob  # type: ignore
 
-        return inst_info_dict
+            instance_pred = convert_to_dst_type(instance_pred, type_pred)[0]
+            pred_type_map = torch.zeros_like(instance_pred)  # type: ignore
+            for key, value in inst_info_dict.items():
+                pred_type_map[instance_pred == key] = value["type"]  # type: ignore
+
+            return pred_type_map, inst_info_dict
+        else:
+            return inst_info_dict
```

Comment:
we won't need else here.
```suggestion
        return inst_info_dict
```